projects/demo/src/modules/components/link/examples/5/index.html

```diff
@@ -0,0 +1,118 @@
+<h3>Default</h3>
```

Comment:
Let's replace 3rd example with this instead. And remove primary on light, it's irrelevant to us as it is the same as primary (it's in the specs for other platforms because it has to be a separate appearance for them due to inability to add tuiTheme="light" on a single element). Also let's add one more entry here at the end with custom color showing that you can write `[style.color]="'var(--tui-status-negative)'"` and override the color.

Reply:
fixed